src/components/MapBox.vue

```diff
@@ -194,25 +194,39 @@
     color: #fff;
   }
 
-  #map {
-    position: absolute;
-    z-index: -1;
-    top: 100px;
-    bottom: 0;
-    width: 100%;
+  .usa-prose{
```

Comment:
Probably won't be an issue here, but since this is a USWD class, there might be side effects from overriding it.